catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-converter/src/test/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/converter/impl/GenericFeatureConverterWfs11Test.java

```diff
@@ -92,7 +92,7 @@ public void testCurveSrs26713() throws IOException {
       assertThat(
           wfc.getFeatureMembers().get(0).getLocation(),
           is(
-              "MULTILINESTRING ((598566.26906782 4914058.52150682, 598557.69477679 4914085.33977038, 598494.67747205 4914254.76475676, 598513.91541973 4914348.11723536, 598593.95441378 4914440.37147034, 598656.43584271 4914500.87603708, 598659.42742456 4914521.61846645, 598652.10605456 4914532.58195028, 598648.43115534 4914544.16261933, 598635.63083991 4914558.16462797, 598554.58586588 4914636.67913744, 598498.3814486 4914752.4469346, 598489.81000042 4914778.04541272, 598421.49284933 4914873.66244196, 598409.19817361 4914931.57794141, 598404.90249957 4914948.64642947, 598340.17638384 4915068.66645978, 598342.49317686 4915118.07263127, 598346.0824185 4915117.02463364, 598566.26906782 4914058.52150682))"));
+              "LINESTRING (598566.26906782 4914058.52150682, 598557.69477679 4914085.33977038, 598494.67747205 4914254.76475676, 598513.91541973 4914348.11723536, 598593.95441378 4914440.37147034, 598656.43584271 4914500.87603708, 598659.42742456 4914521.61846645, 598652.10605456 4914532.58195028, 598648.43115534 4914544.16261933, 598635.63083991 4914558.16462797, 598554.58586588 4914636.67913744, 598498.3814486 4914752.4469346, 598489.81000042 4914778.04541272, 598421.49284933 4914873.66244196, 598409.19817361 4914931.57794141, 598404.90249957 4914948.64642947, 598340.17638384 4915068.66645978, 598342.49317686 4915118.07263127, 598346.0824185 4915117.02463364, 598566.26906782 4914058.52150682)"));
```

Comment:
According to the test file, it is in fact a single linestring. So I think old Geotools was producing the wrong output and we wrote our test around it.

Reply:
This was a bug with the old geotools, but I decided to go back and change the test data instead so that it would be an _actual_ multilinestring and then set this test back to that.